eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -11,7 +11,9 @@ parameters:
   BuildDocs: true
   JobName: Test
   AllocateResourceGroup: 'true'
+  DeployArmTemplate: 'false'
```

Comment:
More of a nit, but is there a reason for the string type instead of boolean here?

Reply:
No there is not, I changed these to be consistent.